src/App.vue

```diff
@@ -329,25 +483,136 @@ export default {
+      this.bigModalTags = image.tags;
+      myModal.show();
+    },
+    closeBigModal() {
+      this.bigModalImage = undefined;
+      this.bigModalTags = undefined;
+    },
+    loadAdminInformation() {
+      const users = [
+        "vmuseum1",
+        "vmuseum2",
+        "vmuseum3",
+        "vmuseum4",
+        "vmuseum5",
+        "vmuseum6",
+        "vmuseum7",
+        "vmuseum8",
+        "vmuseum9",
+        "vmuseum10",
+        "vmuseum11",
+      ];
+      users.forEach((user, index) => {
+        if (this.adminInformation.length !== users.length) {
+          this.adminInformation.push({
+            user: user,
+            todayImage: 0,
+            todayTags: 0,
+            totalImage: 0,
+            totalTags: 0,
+          });
+        }
+        dataFetch.getTodayImagessByUser(user).then((result) => {
+          this.adminInformation[index].todayImage = result.nbImage;
+          this.adminInformation[index].todayTags = result.nbTags;
+          dataFetch.getImagessByUser(user).then((result) => {
+            this.adminInformation[index].totalImage = result.nbImage;
+            this.adminInformation[index].totalTags = result.nbTags;
+          });
         });
+      });
+      this.isLoad = false;
+    },
+    loadImagePerDay() {
+      dataFetch.getTodayImagessByUser(this.user).then((result) => {
+        this.todayImages = result.nbImage;
+        this.todayTags = result.nbTags;
+      });
+    },
+    loadData() {
+      let skipId = undefined;
+      if (this.images) {
+        skipId = this.images[0].element["dcterms:identifier"][0]["@value"];
       }
+      dataFetch.getImages(this.user, skipId).then((result) => {
+        if (result.length === 0) {
+          dataFetch.getSkipImage(this.user, skipId).then((result) => {
+            if (result.length === 0) {
+              if (this.images) {
+                this.nextImage = undefined;
+              } else {
+                this.images = undefined;
+              }
+            } else {
+              if (this.isInitial) {
+                this.selectedTags = [];
+                this.images = result;
+                this.isInitial = false;
+                this.loadData();
+              } else {
+                this.nextImage = result;
+              }
+            }
+            this.isLoad = false;
+          });
+        } else {
+          if (this.isInitial) {
+            this.selectedTags = [];
+            this.images = result;
+            this.isInitial = false;
+            this.loadData();
+          } else {
+            this.nextImage = result;
+          }
+          this.isLoad = false;
+        }
+      });
     },
     removeEntry(position) {
       this.selectedTags.splice(position, 1);
     },
     skipTags() {
-      this.skipId.push(
-        this.images[0].element["dcterms:identifier"][0]["@value"]
-      );
-      this.skipId = [...new Set(this.skipId)];
       this.images[0].element["dcterms:rights"] = [
```

Comment:
why `images` and then using `images[0]`?
also skipId seems only one id, how does it work if multiple skip images?

Reply:
The skipId is used to skip the current image and preload the next image